<launch>
  <node pkg="robot" type="robot_setup.py" name="robot_setup" output="screen"/>
</launch>
